trustmanager/keyfilestore.go

```diff
@@ -9,60 +9,64 @@ import (
 
 const (
 	keyExtension = "key"
+	aliasExtension = "alias"
 )
 
 // KeyStore is a generic interface for private key storage
 type KeyStore interface {
 	LimitedFileStore
 
-	AddKey(name string, privKey data.PrivateKey) error
+	AddKey(name, alias string, privKey data.PrivateKey) error
 	GetKey(name string) (data.PrivateKey, error)
-	AddEncryptedKey(name string, privKey data.PrivateKey, passphrase string) error
-	GetDecryptedKey(name string, passphrase string) (data.PrivateKey, error)
+	GetKeyAlias(name string) (string, error)
```

Comment:
passphraseRetriever needs the context to know what password to ask the user to supply. I don't think we want to have the user to have to enter all of their password or randomly try passwords until one works...